packages/material-ui/src/Select/SelectInput.js

```diff
@@ -264,11 +265,11 @@ const SelectInput = React.forwardRef(function SelectInput(props, ref) {
         )}
         ref={displayRef}
         data-mui-test="SelectDisplay"
-        aria-pressed={open ? 'true' : 'false'}
```

Comment:
`aria-pressed` is for toggle buttons:

> aria-pressed="false" | a | Identifies the button as a toggle button.Indicates the toggle button is not pressed.
> aria-pressed="true" | a | Identifies the button as a toggle button.Indicates the toggle button is pressed.




-- https://www.w3.org/TR/wai-aria-practices/examples/button/button.html